recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - sqlalchemy >=1.4,<3
     - requests >=2.0.0,<3.0.0
     - pyyaml >=5.3
-    - numpy >=1.22.4,<2
+    - numpy >=1.22.4,<3
```

Comment:
This needs to be conditional on the Python version as numpy 2 is only supported for Python >=3.12

Reply:
You mean that langchain itself only supports numpy 2 if python >=3.12 (because numpy 2 is available even on python 3.10 in conda-forge).

Reply:
yes